docs/userguide/datetime/timezone/index.md

```diff
@@ -206,33 +201,41 @@ To do the update, download the .res files appropriate for the platform, as
 described for the .dat file update above, and copy them into the time zone res
 file directory.
 
-#### ICU4C TZ update when ICU is configured for individual files
+### ICU4C TZ update when ICU is configured for individual files
+
+If the ICU-using application sets an ICU data path (or can be changed to set
+one), then the time zone .res file can be placed there. Download the files as
+described above and copy them to the specified directory. See the
+[ICU Data](../../icudata.md) page of the user guide for more information about
+the ICU data path.
 
-#### If the ICU-using application sets an ICU data path (or can be changed to set one), then the time zone .res file can be placed there. Download the files as described above and copy them to the specified directory. See the [ICU Data](../../icudata.md) page of the user guide for more information about the ICU data path.
+### ICU4C TZ update when ICU data is built into a shared library
 
-#### ICU4C TZ update when ICU data is built into a shared library
+1. Set up the environment necessary to rebuild your specific configuration of
+   ICU.
+1. Download the .txt file sources for the updated resources from
+   `http://source.icu-project.org/repos/icu/data/trunk/tzdata/icunew/<IANA tz
+   version>/44`
+1. Copy the downloaded .txt files into the ICU sources for your installation,
+   in the subdirectory source/data/misc/
+1. Rebuid ICU.
+1. Copy the freshly built ICU data shared library to the desired destination.
 
-1.  Set up the environment necessary to rebuild your specific configuration of
-    ICU.
-2.  Download the .txt file sources for the updated resources from
-    http://source.icu-project.org/repos/icu/data/trunk/tzdata/icunew/*<iana tz
-    version number>*/44
-3.  Copy the downloaded .txt files into the ICU sources for your installation,
-    in the subdirectory source/data/misc/
-4.  Rebuid ICU.
-5.  Copy the freshly built ICU data shared library to the desired destination.
+:porint_right: The standard ICU download package contains pre-built ICU data.
```

Comment:
typo in the symbol macro; please copy exactly the prefix from the instructions doc, and use `>` blockquotes for the whole "note" paragraph

Reply:
Fixed typo, and updated all occurrences to use the consistent style as suggested.